SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesModule.h

```diff
@@ -34,4 +37,7 @@ class SPATIALGDKSERVICES_API FSpatialGDKServicesModule : public IModuleInterface
 private:
 	FLocalDeploymentManager LocalDeploymentManager;
 	static FString ParseProjectName();
+
+	static const FString SpatialExe;
```

Comment:
I would remove these from the class and stick them in an anonymous namespace in the cpp file.

Reply:
Doone